### DISCUSSION TITLE
Does kong support proxy dtls protocol and how to configure it?

### DISCUSSION THREAD

**Discussion by Murphy-hub** [Kong/kong#9313] [Help]
I have a business need now.
The upstream service provides the service of udp and needs to provide the service of the dtls protocol through the gateway. I tried to configure the routing protocol as tls and the service protocol as udp, but it did not work as expected.
What should I do, thanks.

**Comment by fffonion** [comment]
Hi @Murphy-hub , dtls is not supported currently. There are some patches available http://nginx.org/patches/dtls/README.txt, you might use that to build kong using https://github.com/kong/kong-build-tools.
